internal/slicer/slicer_test.go

```diff
@@ -769,6 +769,79 @@ var slicerTests = []slicerTest{{
 	report: map[string]string{
 		"/dir/nested/file": "file 0644 84237a05 {test-package_myslice}",
 	},
+}, {
+	summary: "Generate (chisel-state) does not impact the fs",
```

Comment:
Can we merge this test with another one? I think it would be useful to:
* add chisel-state to another basic test just to see that nothing is changing.
* add a specific test to see that the parent folder permission is respected (and possible db generated but it depends on how the code is laid out).

Reply:
Done! 3af168d